backend/src/main/java/hu/flowacademy/worksheet/service/KeycloakClientService.java

```diff
@@ -66,7 +56,7 @@ public int createAccount(String name, String email) throws WorksheetUserExceptio
         return HttpStatus.CREATED.value();
     }
 
-    private static String[] nameChecker(String name) {
+    private String[] nameChecker(String name) {
```

Comment:
Erre feltétlenül szükség van? Nem tudna a `createAccount` külön fist name, last name párost kapni, illetve a frontendtől a DTO is? Problémásak lehetnek az olyan nevek ahol több keresztnév vagy vezetéknév van.